plugins/src/main/java/tech/pegasys/pantheon/plugins/services/PicoCLIOptions.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.plugins.services;
+
+/** This service will be available during the registration callbacks. */
+public interface PicoCLIOptions {
```

Comment:
Just checking my understanding here.  Seems like the idea in tying this class to PicoCLI rather than making it something more generic like `CLIOptions` is that in the future we might use a different CLI system, that will be supported by a completely different service / api?

Reply:
+1, strongly coupled to Pico cli library.

Reply:
Yes, graceful migration.